cdap-ui/app/cdap/components/Header/Welcome/Welcome.scss

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$title-color: white;
+$header-height: 150px;
+
+.cdap-body-container .modal.fade.show {
+  .welcome-modal.modal-dialog {
+    transform: translateY(-50%);
+    margin-top: 50vh;
```

Comment:
I believe this is the cause for the modal to appear from the bottom as it get the margin top of 50vh and a translate property applied. The existing transition manipulates translate css to render it from top. This will cause this modal alone to appear from bottom.

Reply:
It's actually the transform property that's causing that. I removed it and just apply less margin-top